todocore/src/main/java/com/knighttodo/todocore/service/DayService.java

```diff
@@ -53,4 +54,12 @@ public void deleteById(UUID dayId) {
         dayRepository.softDeleteAllDayTodosByDayId(dayId);
         dayRepository.softDeleteById(dayId);
     }
+
+
+    public DayVO findDayByDate(LocalDate date) {
+        return dayRepository.findDayByDate(date).map(dayMapper::toDayVO).orElseThrow(() -> {
```

Comment:
I suppose here you need to use findDayByDateisAlive(date), because the project used soft delete. Don't forget to add @Query where removed=false.